server/db/invalid_event.go

```diff
@@ -0,0 +1,95 @@
+package db
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type InvalidEventType string
```

Comment:
Do you need these two structs? `InvalidEventOverview` and `InvalidEventProperty`